monitoring/mock_uss/scdsc/routes_injection.py

```diff
@@ -118,7 +117,7 @@ def inject_flight(flight_id: str) -> Tuple[str, int]:
     record = database.FlightRecord(
         op_intent_reference=result.operational_intent_reference,
         op_intent_injection=req_body.operational_intent,
-        flight_authorisation=req_body.flight_authorisation)
+        flight_authorisation=req_body.flight_authorisation if 'flight_authorisation' in req_body else None)
```

Comment:
nit: if flight_authorisation is made non-optional per comment above, then this change isn't needed either

Reply:
Done thanks!